front/src/applications/stdcm/types.ts

```diff
@@ -54,6 +55,35 @@ export type StdcmPathProperties = {
   zones: NonNullable<PathProperties['zones']>;
 };
 
+export type RefSchedule = {
```

Comment:
I think you don't need this type and can use `RefScheduleWithSecondaryCode` everywhere